### ISSUE TITLE
Raspberry Pi?

### ISSUE THREAD

**Issue by blakkkk** [Uuvana-Studios/longvinter-linux-server#20]
Is it possible to run this server on a Raspberry Pi? I have tried almost everything and get stuck on installing lib32stdc++6. I'm assuming it's because the Raspberry Pi processor is arm64.

**Comment by denizkoekden** [comment]
Yes. The Games does not work on ARM. Not yet. IDK if it will be possible for the devs to compile an ARM Version.

**Comment by DJTOMATO** [comment]
Neither works on Aarch64 =( such a shame
